.github/workflows/build.yaml

```diff
@@ -0,0 +1,38 @@
+name: "build"
+on: [push, pull_request]
+env:
+  TRIVY_VERSION: 0.43.1
+  COPA_VERSION: 0.3.0
+permissions: write-all
```

Comment:
why do we need write-all?

Reply:
I thought the test wasn't working since it couldn't load the docker image or trivy json file but it didn't help that issue. Will change this to read-all.